telegram/ext/jobqueue.py

```diff
@@ -372,20 +373,22 @@ def __init__(self,
                  repeat=True,
                  context=None,
                  days=Days.EVERY_DAY,
+                 tzinfo=UTC,
```

Comment:
Please move `tzinfo=UTC` to be the last kwarg, in order to maintain backward compatibility.

Reply:
Fixed in 6ed29bb